package.json

```diff
@@ -8,6 +8,19 @@
     "icons"
   ],
   "main": "index.js",
+  "exports": {
+    ".": {
+      "module": "./index.js",
```

Comment:
Module condition is documented [here](https://webpack.js.org/guides/package-exports/#providing-commonjs-and-esm-version-stateless) (webpack docs).
> The module condition is used as optimization when preprocessing node-targeted code with a tool that supports ESM for require() (like a bundler, when bundling for Node.js). For such a tool the exception is skipped. This is technically optional, but bundlers would include the package source code twice otherwise.

I'm not exactly sure what this does, but it seems to be an optimization nonetheless.

Reply:
Thank you for sharing the docs, that's very helpful. One question though, based on both this and Node's docs, shouldn't the keys and values be like the following:

```json
{
"module": "./icons.mjs",
"import": "./icons.mjs",
"require": "./icons.js",
"default": "./icons.js"
}
```

Or potentially the following if we want something shorter (based on what's in the webpack docs)

```json
{
  "module": "./icons.mjs",
  "require": "./icons.js",
}
```

Though my preference goes to the first, as I'm not sure we can omit the "import" field since the "module" fields isn't mentioned in the Node docs :thinking:

Reply:
"require" is not necessary with "default," but we can include it if you want to be more explicit (see https://github.com/simple-icons/simple-icons/pull/6189#issuecomment-898907491). We cannot omit the "import" field, Node does not resolve from the "module" field.